web/beacon-app/src/features/auth/types/RegisterService.ts

```diff
@@ -25,7 +25,10 @@ export type NewUserAccount = Omit<
   'user_id' | 'name' | 'organization' | 'domain' | 'terms_agreement' | 'privacy_agreement'
 >;
 
-export type NewInvitedUserAccount = Omit<User, 'user_id' | 'organization' | 'domain'>;
+export type NewInvitedUserAccount = Omit<
+  User,
+  'user_id' | 'name' | 'organization' | 'domain' | 'terms_agreement' | 'privacy_agreement'
```

Comment:
```suggestion
export type NewInvitedUserAccount = Pick<User, 'password' | 'pwcheck' | 'email'>;
```

- Use `Pick` when you want only a few properties from a long list. For instance, if you need to select 2 values from a type containing 5 properties, `Pick` would be the best choice.

- Use `Omit` when it's more practical to exclude a few properties. Suppose you have a type with 9 properties, and you require 6 of them while excluding 3. In such cases, `Omit`  is the choice, as it helps you exclude the 3 values you don't need.

Reply:
To be honest, I copied what was on a line above for a `NewUser` instead of using `Pick`. I was being lazy since I knew this will change soon. I do appreciate the feedback and suggestion.